go.mod

```diff
@@ -39,3 +40,5 @@ require (
 	k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
 	k8s.io/utils v0.0.0-20191010214722-8d271d903fe4 // indirect
 )
+
+replace github.com/layer5io/service-mesh-performance-specification v0.0.0-20200617124327-27b13cd1be5e => github.com/kanishkarj/service-mesh-performance-specification v0.0.0-20200617164720-8232593512c6
```

Comment:
Is this for testing?

Reply:
Oh yes, I have some changes up in my repository. So it's because of that.. I'll remove this once we have those changes merged.